Public/Sdk/Public/Managed/helpers.dsc

```diff
@@ -132,4 +136,44 @@ namespace Helpers {
 
         return results.toArray();
     };
+
+    /** TODO: Unofortunately the System.Interactive.Async package exposes IAsyncEnumerable in its own System.Generic.Collection namespace,
+    /*        colliding with .NETCore 3.0 implementation of the same name. For the time being (hopefully once the package maintainers updated their deps),
+    /*        we explicitly create an aliased reference for the code that depends on this.
+    **/
+
+    @@public
+    export function aliasedReferencesForSystemInteractiveAsync(references: Shared.Reference[]) : Csc.Arguments {
+        const needsSystemInteractiveAsync =
+            references.some(ref => (ref as Shared.ManagedNugetPackage).name !== undefined &&

```

Comment:
>(ref as Shared.ManagedNugetPackage [](start = 35, length = 34)

I beleive we have helper conditions:
`Shared.isManagedPackage(ref)` which will do the type coecion for you so you can write:
`Shared.isManagedPackage(ref) && ref.name.contains("....")`

so you don't have to do tostring either becaues nuget packages are always string for name.